### DISCUSSION TITLE
CSRF token

### DISCUSSION THREAD

**Discussion by Artanisky** [garethredfern/laravelvue-spa#9] [Q&A] [answered]
Hi I have a problem with CSRF. When I try to send a request, the api gives the following error
![image](https://user-images.githubusercontent.com/32029918/128608330-8caf3e04-f240-4906-90b2-268547969fb1.png)
P.S Didn't make any changes in the demo project(except for installing the database and url).

**Comment by 0xgdr** [comment, marked as answer]
Hi there, did you make sure to set up your endpoints in the Laravel API  .env file - make sure to change them to whatever you are using in your local setup:

```
SANCTUM_STATEFUL_DOMAINS=localhost:8080
SPA_URL=http://localhost:8080
SESSION_DOMAIN=localhost
```

See https://laravelvuespa.com/setup/laravel-setup/#install-sanctum

**Comment by ferlaeng** [comment]
i-ve resolve add 
SESSION_DRIVER=cookie
SESSION_DOMAIN='.localhost'
